src/pages/HomePage/components/Products/Products.jsx

```diff
@@ -87,16 +139,10 @@ function Products() {
                 <span className={styles["item-related"]}>plan tree</span>
                 <span className={styles["item-related"]}>david backben</span>
             </div>
-            <div className={styles["list-products"]}>
-                {
-                    listProducts.map((item, index) => (
-                        <ProductItem
-                            item={item}
-                            key={index}
-                        />
-                    ))
-                }
-            </div>
+            {
+                viewMode ? <ListView /> : <GridView />
```

Comment:
```suggestion
                activeViewMode  === 'List' ? <ListView /> : <GridView />
```
viewMode ở đây sử dụng cái activeViewMode luôn thay vì sử dụng giá trị true false nhé